app/src/main/java/com/mk/tasky/agenda/data/AgendaRepositoryImpl.kt

```diff
@@ -238,8 +239,35 @@ class AgendaRepositoryImpl(
         }
     }
 
+    private suspend fun deleteEventLocaly(id: String) {
+        dao.deleteEventById(id)
+        dao.deleteEventCrossRefById(id)
+    }
+
+    override suspend fun deleteEventById(id: String) {
+        val event = dao.getEventById(id).toDomain()
+        alarmRegister.cancelAlarm(event)
+
+        supervisorScope {
+            val localJob = launch { deleteEventLocaly(id) }
+
+            val remoteJob = launch {
+                deleteEventByIdRemotely(id).onFailure {
+                    println(it.message)
```

Comment:
I'd rather use something like `Timber` for logging. That makes it easier to hide logs in the release build.

Reply:
Yeah that's a goal for the future. Right now I use "print" so I can put breakpoints and such